### ISSUE TITLE
Missing Codelyzer (Angular) rule: template-accessibility-tabindex-no-positive

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#497]
### Missing Converter

-   `tslint-to-eslint-config` version: 1.0.0-beta

-   `typescript-eslint` version: 2.31.0

#### TSLint Rule Name

template-accessibility-tabindex-no-positive

#### Package Source

**Codelyzr**: http://codelyzer.com/rules/template-accessibility-tabindex-no-positive

Splitting out from #421, this issue tracks adding in a missing Codelyzr rule converter.

**Comment by JoshuaKGoldberg** [comment]
Closing as [angular-eslint](https://github.com/angular-eslint/angular-eslint/) does not yet have an equivalent for this rule. 💀 

If you're seeing this and they've since added one, hooray! Post here and we can re-open it. Feel free to send a pull request resolving this issue.

**Comment by andrefilimono** [comment]
Hi, the rule is available since v0.4.0-beta.1, but the name has changed to `no-positive-tabindex`.

https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/rules/no-positive-tabindex.ts
